src/app/report/edit-display/encoding-button/encoding-button.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { ReportOrCheckpoint, ReportUtil } from '../../../shared/util/report-util';
+import { ToastService } from '../../../shared/services/toast.service';
+import { CustomEditorComponent } from '../../../custom-editor/custom-editor.component';
+
+@Component({
+  selector: 'app-encoding-button',
+  standalone: true,
+  imports: [],
+  templateUrl: './encoding-button.component.html',
+  styleUrl: './encoding-button.component.css',
+})
+export class EncodingButtonComponent implements OnChanges {
+  @Input({ required: true }) selectedNode!: ReportOrCheckpoint;
+  @Input({ required: true }) editor!: CustomEditorComponent;
+  buttonType: string = 'Base64';
```

Comment:
Does it always start at Base64?

Reply:
Yes, if the encoding is set to base64 it will always start as base64